### PR TITLE
Serialization_Mask_Disable

### PR DESCRIPTION
**Pull request by stitova-idm**
Turning off masking ability.

### CODE REVIEW

File: Eradication/LarvalHabitatMultiplier.cpp [hunk 1]
```diff
@@ -75,7 +75,19 @@ namespace Kernel
...
+        }
+
+        if( !m_Multiplier.at( vht ).count( species ) )
+        {
+            release_assert( false );
+        }
+        else
+        {
+            return m_Multiplier.at( vht ).at( species );
+        }
```
Comment (Bridenbecker) [inline Eradication/LarvalHabitatMultiplier.cpp:90]: Did you always have this method this way?  I would swear you had line 85 returning 1.  However, I like this.  You could make it cleaner and look something like:

release_assert( m_Multiplier.count( vht ) > 0 );
release_assert( m_Multiplier.at( vht ).count( species ) > 0 );
return m_Multiplier.at( vht ).at( species );

File: Eradication/SerializationParameters.cpp [hunk 2]
```diff
@@ -148,15 +148,24 @@ std::deque<int32_t> SerializationParameters::ProcessConfig( std::vector<int32_t>
 
 void SerializationParameters::CheckSupportedFlags( SerializationBitMask_t serialization_mask, std::string& parameter ) const
 {
+    // Commenting it out for now so can be used later when we re-implement the bitmasking
```
Comment (Bridenbecker) [inline Eradication/SerializationParameters.cpp:151]: How about referring to the ticket as the reason for commenting it out?

**Review by Bridenbecker — APPROVED** [review]
I would appreciate the comment in CheckSupportFlags() that reference the ticket about the feature not working.

File: Eradication/LarvalHabitatMultiplier.cpp (hunk 1 above)
Comment (stitova-idm) [reply Eradication/LarvalHabitatMultiplier.cpp:90]: it was returning 1 as the "fix" for the m_Multiplier being initialized incorrectly, but I changed it so now it's a release assert if it gets here and it's not initialized properly.

File: Eradication/LarvalHabitatMultiplier.cpp [hunk 3]
```diff
@@ -75,6 +75,8 @@ namespace Kernel
 
     float LarvalHabitatMultiplier::GetMultiplier( VectorHabitatType::Enum vht, const std::string& species ) const
     {
+        release_assert( m_Multiplier.count( vht ) > 0 );
+        release_assert( m_Multiplier.at( vht ).count( species ) > 0 );
```
Comment (Bridenbecker) [inline Eradication/LarvalHabitatMultiplier.cpp:79]: nice

**Review by Bridenbecker — APPROVED** [review]
LGTM.  Just make sure to run test to make sure everything still passes.

**Pull request merged**